analyzer/runtime/runtime.go

```diff
@@ -304,7 +304,7 @@ func (m *Main) queueDbUpdates(batch *storage.QueryBatch, data *BlockData) {
 			transactionData.Index,
 			transactionData.Hash,
 			transactionData.EthHash,
-			transactionData.Fee.String(),
+			&transactionData.Fee, // pgx bug? Needs a *BigInt (not BigInt) to know how to serialize.
```

Comment:
maybe we defined a method only on *BigInt?

Reply:
v tangential but `encoding/gob` complains if you use pointer receivers like `func (t *MyType) MarshalBytes`... go pointers are funky i guess